Feat/new tags

valeryV99 opened the pull request:
https://www.notion.so/Create-tags-for-the-testnet-swap-dapp-26a362757c9880098865db43a68fe4c8?v=254362757c98811fb5a9000c47b44575

valeryV99 committed:
feat: add token tags and improve TokenName component

- Introduced `tags` property in `Token` type and populated token list with community tags.
- Updated `TokenName` component to display tags with SVG icons (`community-tag` and `bag-wars-tag`).
- Modified `TokenSelect` and `TokenName` to support `showTags` prop for rendering tags.
- Added new SVG assets for token tags (`community-tag.svg` and `bag-wars-tag.svg`).

valeryV99 committed:
feat: enhance TokenName component and add tests

- Updated `TokenName` to fix `key` and `src` assignment for tag rendering.
- Enabled `globals` in Vitest configuration for simplified testing.
- Added comprehensive unit tests for `TokenName` to validate rendering logic and props handling.
- Refactored test setup with mocks for `useToken` and `useTokenList`.

opcode0x commented on apps/web/src/features/token/ui/token-name/TokenName.tsx:
```diff
@@ -47,15 +63,30 @@ export const TokenName = ({
-          >
-            {rune?.spaced_name ?? name}
-          </span>
+          <>
```
no need to wrap with fragment

valeryV99 committed:
feat: add reusable Tooltip component

- Introduced a customizable `Tooltip` component using `@radix-ui/react-tooltip`.
- Added support for delay, side positioning, hoverable content, and styling via `TooltipProps`.
- Exported auxiliary Tooltip subcomponents for extended use cases.
- Updated `pnpm-lock.yaml` to include new `@radix-ui` dependencies.
